pages/common/rename.md

```diff
@@ -0,0 +1,30 @@
+# rename
+
+> Rename a file or group of files with a regular expression.
+> WARNING: This command will overwrite files without prompting unless the dry-run option is used.
+> Note: This page refers to the Perl version, also known as `file-rename`.
+> More information: <https://manned.org/file-rename>.
```

Comment:
```suggestion
> More information: <https://manned.org/man/prename>.
```
Since there's two different perl renames, lets link to this instead. That way you can see in the sidebar that there's `1` and `1p`

Reply:
Don't. Preserve the commit history so that it's visible what commit was made after what comment. We'll squash before merge so a messy PR doesn't matter.

Reply:
> I just did one last change, see if it makes sense: the one file we left as the basis (common/rename.md) ended up with the external reference to https://manned.org/prename, which is a bit funny, since the basic form is known in the Perl community as "file-rename". Since "prename" is more like an alias, I reverted that to reference https://manned.org/file-rename instead, which seems more consistent to me. If you follow the link to both, you will see the text referencing "file-rename".

Yes, but I requested using prename for the reasons highlighted above.